lib/galaxy/dependencies/pipfiles/default/pinned-dev-requirements.txt

```diff
@@ -34,6 +34,7 @@ pyparsing==2.3.1
 pytest-html==1.20.0
 pytest-metadata==1.8.0
 pytest-pythonpath==0.7.3
+pytest-postgresql==1.3.4
```

Comment:
You need to add this to `lib/galaxy/dependencies/pipfiles/default/Pipfile`

Reply:
Thanks @nsoranzo, I've rebased that into https://github.com/galaxyproject/galaxy/pull/7548/commits/e7e8283560ef061450a96c96d7cb2f61a6d41ba4